packages/holocron-dev-server/src/components/Document.jsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { bundleType as defaultBundleType } from '../constants';
+import { getPublicAppUrl } from '../utils';
+
+/* eslint-disable react/prop-types */
+export default function Document({
+  bundleType = defaultBundleType,
+  scripts = [],
+  moduleMap = {},
+  initialState = {},
+  lang = 'en-US',
+}) {
+  return (
+    <html lang={lang}>
+      <head>
+        <title>Holocron dev server</title>
```

Comment:
We should mention that this server is different from one app and we don't do SSR

Reply:
Will update the docs.